.github/workflows/master.yml

```diff
@@ -9,148 +9,233 @@ on:
     - master
```

Comment:
Oh, BTW, @dongjoon-hyun. I happened to remove the JDK 11 builds here. I filed a JIRA to recover it at SPARK-32248. I promise I will take a look right away after this PR is merged.